sdk/include/reactos/drivers/bootvid/display.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * PROJECT:     ReactOS video boot driver
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Provides display settings for BOOTVID, NTOSKRNL and HAL
+ * COPYRIGHT:   Copyright 2020 Dmitry Borisov (di.sean@protonmail.com)
+ */
+
+#pragma once
+
+/* For default VGA */
+#define SCREEN_WIDTH    640
+#define SCREEN_HEIGHT   480
+
+#define VID_SCROLL_AREA_LEFT       32
```

Comment:
Are you going to use these VID_ defines elsewhere than in NTOSKRNL? If not I would strongly suggest you move them back to some header in ntoskrnl/inbv.

Reply:
Yes moving this header to inbv would be better. It looks a bit strange when kernel and HAL import header from some driver, and not vice versa. :)